tests/unit/services/test_provider.py

```diff
@@ -237,7 +237,7 @@ def test_get_lxd_provider(monkeypatch, provider_service, lxd_remote, check):
     check.equal(actual, mock_provider.return_value)
     with check:
         mock_provider.assert_called_once_with(
-            lxd_project="testcraft", lxd_remote=lxd_remote
+            lxd_project="testcraft", lxd_remote=lxd_remote, intercept_mknod=True
```

Comment:
Can you exercise the default value and setting `ProviderService(intercept_mknod)`?

A quick way to do this would be to instantiate a ProviderService like some of the other tests and add something along the lines of `@pytest.param("intercept_mknod", [None, True, False])`